Source/UI/NewUI/BillingForm/ViewModel/DefaultBillingFormViewModel.swift

```diff
@@ -74,6 +74,9 @@ final class DefaultBillingFormViewModel: BillingFormViewModel {
             textValueOfCellType[type.index] = hasValue ? value : (!isMandatory ? "" : nil)
             errorFlagOfCellType[type.index] = hasValue ? false : isMandatory
         }
+        if let country = country, country.iso3166Alpha2.isEmpty {
+            self.country = Country(iso3166Alpha2: Locale.current.regionCode ?? "", dialingCode: nil)
```

Comment:
feels like a complicated way of doing it. Couldn't we just target the initialiser directly?

```
    public init(iso3166Alpha2: String, dialingCode: String?) {
        self.iso3166Alpha2 = iso3166Alpha2.isEmpty ? Locale.current.regionCode : iso3166Alpha2
        self.dialingCode = dialingCode
    }
```

We can then just test this simply with Country initialiser unit test.

Reply:
The reason I say this feels complicated is that it feels a little hacky, after setup to check if something exists, then if it exists but lacks something, to override.

Reply:
Ok makes sense for now we should ideally be making these changes in checkout iOS SDK if needed. but for now I am fine with this.